R/html_dependency.R

```diff
@@ -231,32 +231,37 @@ copyDependencyToDir <- function(dependency, outputDir, mustWork = TRUE) {
     }
   }
 
-  if (!file.exists(outputDir))
+  if (!dir_exists(outputDir))
     dir.create(outputDir)
 
   target_dir <- file.path(outputDir,
     paste(dependency$name, dependency$version, sep = "-"))
 
-  if (!file.exists(target_dir)) {
-    dir.create(target_dir)
+  if (dir_exists(target_dir))
+    unlink(target_dir, recursive = TRUE)
```

Comment:
This scares me - if you accidentally specify the wrong target, you're going to wipe out a directory